techlibs/xilinx/synth_xilinx.cc

```diff
@@ -49,10 +49,25 @@ struct SynthXilinxPass : public ScriptPass
 		log("    -top <module>\n");
 		log("        use the specified module as top module\n");
 		log("\n");
-		log("    -family {xcup|xcu|xc7|xc6v|xc5v|xc6s}\n");
+		log("    -family <family>\n");
 		log("        run synthesis for the specified Xilinx architecture\n");
 		log("        generate the synthesis netlist for the specified family.\n");
-		log("        default: xc7\n");
+		log("        supported values:\n");
+		log("        - xcup: Ultrascale Plus\n");
+		log("        - xcu: Ultrascale\n");
+		log("        - xc7: Series 7\n");
+		log("        - xc6s: Spartan 6\n");
+		log("        - xc6v: Virtex 6\n");
+		log("        - xc5v: Virtex 5 (EXPERIMENTAL)\n");
+		log("        - xc4v: Virtex 4 (EXPERIMENTAL)\n");
+		log("        - xc3sda: Spartan 3A DSP (EXPERIMENTAL)\n");
+		log("        - xc3sa: Spartan 3A (EXPERIMENTAL)\n");
+		log("        - xc3se: Spartan 3E (EXPERIMENTAL)\n");
+		log("        - xc3s: Spartan 3 (EXPERIMENTAL)\n");
+		log("        - xc2vp: Virtex 2 Pro (EXPERIMENTAL)\n");
```

Comment:
Also for Virtex 2?

Reply:
Virtex 2 should be identical to Virtex 2 Pro wrt synthesis (they differ by supported I/O standards and by the presence of gigabit transceivers and PPC cores), but I'd rather accept both values as valid for consistency.  Also, if we ever get timing information for them, it's likely to be different.